frontend/tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
     "lib": ["dom", "dom.iterable", "esnext"],
+    "paths": {
+      // "~/*": ["src/*"] // Potentially create this path aliases
```

Comment:
@nathanrydel, please let me know if this is something you are open to implement, or I can remove this line all together,
thanks

Reply:
It will clean up the imports in other files/dirs, so I am OK with implementing it.

Thank you